pkg/cmd/extension/command.go

```diff
@@ -219,6 +221,33 @@ func NewCmdExtension(f *cmdutil.Factory) *cobra.Command {
 				return nil
 			},
 		},
+		&cobra.Command{
+			Use:   "run <name> [args]",
```

Comment:
Let's rename this command to `exec`. The name `run` is also fine, but my teammates and I felt that `exec` is a little bit closer to what's going on here. Also, "exec" is a little bit more unusual verb than "run", suggesting that this is a lower-level command that should only be used in special occasions.